images/linux/scripts/helpers/install.sh

```diff
@@ -152,6 +152,41 @@ get_github_package_hash() {
     echo "$result"
 }
 
+download_hash_from_file() {
+    local url=$1
+    local keyword=$2
+    local optional_keyword=$3
```

Comment:
Take a look at the new order of things here. I think looks better now. Still not sure about renaming variable.

Reply:
I still didn't get the idea of multiple regular expressions, but if you think it's handy, ok then.

Reply:
@vpolikarpov-akvelon  ~~I'm afraid of regular expressions~~ Because of words order. This way it will work with any combination - a little help for future me.